addons/license-v2.yml

```diff
@@ -96,8 +100,23 @@ actions:
           modules = isAdc ? ("${settings.modules}" == "true" ? "litemageunlimited":"cache") : "${settings.modules:cache}",
           workers = ${settings.workers:1},
           domains = ${settings.domains:0},
+          litemage = ${settings.lm:"false"},
           cloudlets = getParam("cloudlets") || ${nodes.cp.master.flexibleCloudlets:0};
       if (!domains) domains = (workers == 1 && cloudlets <= 16) ? 1 : 5;
+      if (litemage == "false") {
+          litemage = "OFF"
+      } else {
+          litemage = "ON"
+      };
+      var resp = jelastic.env.control.AddContainerEnvVars({
+          envName: "${env.name}",
+          session: session,
+          nodeGroup: "${globals.lsNodeGroup}",
+          vars: {
+            "LITEMAGE": litemage
```

Comment:
`"LITEMAGE": (litemage == "false") ? "OFF" : "ON"`

ternary operator would be more readable in this case